src/v2/Apps/Settings/Routes/EditSettings/SettingsEditSettingsRoute.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { SettingsEditSettingsRoute_me } from "v2/__generated__/SettingsEditSettingsRoute_me.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SettingsEditSettingsInformationFragmentContainer } from "./SettingsEditSettingsInformation"
+import { Column, GridColumns, Join, Separator } from "@artsy/palette"
+
+interface SettingsEditRouteProps {
+  me: SettingsEditSettingsRoute_me
+}
+
+const SettingsEditRoute: React.FC<SettingsEditRouteProps> = ({ me }) => {
+  return (
+    <GridColumns>
+      <Column span={8}>
+        <Join separator={<Separator my={4} />}>
+          <SettingsEditSettingsInformationFragmentContainer me={me} />
+
+          {/* TODO: SettingsEditSettingsPassword */}
+
+          {/* TODO: SettingsEditSettingsTwoFactor */}
+
+          {/* TODO: SettingsEditSettingsLinkedAccounts */}
+
+          {/* TODO: SettingsEditSettingsEmailPreferences */}
+
+          {/* TODO: SettingsEditSettingsDeleteAccount */}
```

Comment:
So here you can see the components left on this page. We should follow this naming convention which prevents collisions. Relay related code needs to have globally unique names.